nncf/version.py

```diff
@@ -11,6 +11,6 @@
 
 __version__ = "2.6.0"
 
-BKC_TORCH_VERSION = "2.0.1"
-BKC_TORCHVISION_VERSION = "0.15.1"
+BKC_TORCH_VERSION = "2.1.1"
```

Comment:
Please add row "develop" to https://github.com/openvinotoolkit/nncf/blob/develop/docs/Installation.md#corresponding-versions 
with new versions of pt.

Reply:
Done